src/gui/ContextMenus.h

```diff
@@ -336,6 +356,16 @@ class NodePopupMenu : public PopupMenu
                     break;
             }
         }
+        else if (result >= 40000 && result < 50000)
+        {
+            const int osFactor = (int) powf(2, float (result - 40000));
+            if (auto gNode = node.getGraphNode())
+            {
+                gNode->setOversamplingFactor (osFactor);
+                gNode->getParentGraph()->releaseResources();

```

Comment:
Probably ok for now, but this might cause problems with graphs with heavy weight plugins loaded.  e.g. Kontakt or anything else that takes forever to load.  Nice work!